src/components/element-editor/basic-details/basic-details.tsx

```diff
@@ -38,52 +38,91 @@ export const BasicDetails = () => {
     // =========================================================================
 
     return (
-        <div>
-            <Form.Input
-                {...register("id")}
-                required
-                label="ID"
-                placeholder="Enter ID"
-                errorMessage={errors.id?.message}
+        <FieldEditorAccordionItem type="default" expanded title="Basic">
+            <Controller
+                name="type"
+                control={control}
+                render={() => (
+                    <IconDropdown type={element?.type} id={element?.id} />
```

Comment:
need to integrate dropdown menu with react-hook-form too. as this is a custom developed component, there is a need to fire the onChange event manually to pass the value back to react-hook-form

Reply:
i created a function to pass the value from the custom dropdown up the basic details component, but since the updating of the dropdown value is not covered in this ticket i have just made the fundtion print out the updated option

![image](https://github.com/LifeSG/web-form-builder/assets/103546625/93ee8178-4201-43b5-a47f-40e8b182a303)